apps/demos/Demos/Chat/Customization/jQuery/styles.css

```diff
@@ -0,0 +1,52 @@
+.dx-viewport {
+    padding: 20px;
+    min-width: 780px;
+    border: 1px solid var(--dx-color-border);
+    border-radius: 8px;
+}
+
+.demo-container {
+    min-width: 720px;
+    display: flex;
+    gap: 20px;
+}
+
+.chat-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    align-items: center;
+    justify-content: center;
+}
+
+.options {
+    padding: 20px;
+    min-width: 280px;
+    background-color: rgba(191, 191, 191, 0.15);
+}
+
+#chat {
+max-width: 480px;
+}
+
+.caption {
+    font-size: 18px;
+    font-weight: 500;
+}
+
+.option-group {
+    padding: 10px 0;
+    border-bottom: 1px solid lightgray;
+}
+
+.option-group:last-child {
+    border-bottom: none;
+}
+
+.option {
+    margin: 20px 0;
```

Comment:
Kindly check the Figma prototype and set the correct values between elements. I notice there are many differences between your implementation and the Figma design

<img width="277" alt="image" src="https://github.com/user-attachments/assets/a15686a7-0501-4ea4-b69c-76748e5881bd">
<img width="274" alt="image" src="https://github.com/user-attachments/assets/9abe5dc4-84e6-487d-88a7-8bad0d576116">

Reply:
https://github.com/DevExpress/DevExtreme/pull/28441/commits/7abeedba5198baaf75acd81fdb9e03e779b24d94